config/example/node.yaml

```diff
@@ -1,159 +1,159 @@
+  level: debug  # logger level
 
 profiler:
-  address: 127.0.0.1:6060
-  shutdown_timeout: 15s
+  address: 127.0.0.1:6060  # endpoint for Node profiling
+  shutdown_timeout: 15s  # timeout for graceful profiling HTTP server shutdown
 
 metrics:
-  address: 127.0.0.1:9090
-  shutdown_timeout: 15s
+  address: 127.0.0.1:9090  # endpoint for Node metrics
+  shutdown_timeout: 15s  # timeout for graceful metrics HTTP server shutdown
 
 node:
-  key: ./wallet.key
+  key: ./wallet.key  # path to a binary private key
   wallet:
-    path: "./wallet.json"
-    address: "NcpJzXcSDrh5CCizf4K9Ro6w4t59J5LKzz"
-    password: "password"
-  addresses:
+    path: "./wallet.json"  # path to a NEO wallet; ignored if key is presented
+    address: "NcpJzXcSDrh5CCizf4K9Ro6w4t59J5LKzz"  # address of a NEO account in the wallet; ignored if key is presented
+    password: "password"  # password for a NEO account in the wallet; ignored if key is presented
+  addresses:  # addresses for bootstrapping in the Network map
     - s01.neofs.devenv:8080
     - /dns4/s02.neofs.devenv/tcp/8081
     - grpc://127.0.0.1:8082
     - grpcs://localhost:8083
   attribute_0: "Price:11"
   attribute_1: UN-LOCODE:RU MSK
-  relay: true
-  persistent_state:
+  relay: true  # (not)register Node in the Network map on startup
+  persistent_state:  # path to store Node state persistently
     path: /state
 
 grpc:
   num: 2
   0:
-    endpoint: s01.neofs.devenv:8080
+    endpoint: s01.neofs.devenv:8080  # endpoint for gRPC server
     tls:
-      enabled: true
-      certificate: /path/to/cert
-      key: /path/to/key
+      enabled: true  # use TLS for a gRPC connection
+      certificate: /path/to/cert  # path to TLS certificate
+      key: /path/to/key  # path to TLS key
 
   1:
-    endpoint: s02.neofs.devenv:8080
+    endpoint: s02.neofs.devenv:8080  # endpoint for gRPC server
     tls:
-      enabled: false
+      enabled: false  # use TLS for a gRPC connection
 
 control:
-  authorized_keys:
+  authorized_keys:  # list of raw hex public keys that have rights to use the Control Service
     - 035839e45d472a3b7769a2a1bd7d54c4ccd4943c3b40f547870e83a8fcbfb3ce11
     - 028f42cfcb74499d7b15b35d9bff260a1c8d27de4f446a627406a382d8961486d6
-  grpc:
+  grpc:  # endpoint that are listened by the Control Service
     endpoint: 127.0.0.1:8090
 
-contracts:
+contracts:  # side chain NEOFS contract script hashes
   balance: 5263abba1abedbf79bb57f3e40b50b4425d2d6cd
   container: 5d084790d7aa36cea7b53fe897380dab11d2cd3c
   netmap: 0cce9e948dca43a6b592efe59ddb4ecb89bdd9ca
   reputation: 441995f631c1da2b133462b71859494a5cd45e90
   proxy: ad7c6b55b737b696e5c82c85445040964a03e97f
 
 morph:
-  dial_timeout: 30s
-  disable_cache: true
-  rpc_endpoint:
+  dial_timeout: 30s  # timeout for side chain NEO RPC client connection
+  disable_cache: true  # do not use TTL cache for side chain GET operations
+  rpc_endpoint:  # side chain NEO RPC endpoints; are shuffled and used one by one until the first success
```

Comment:
shuffled or round-robin?

Reply:
they are shuffled and 2nd(in the shuffled list) is not going to be used(even connection is not established) until the first one returns error and so on